oidc-client-app-samples/browser-client-app-sample/app/index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<!--
+~ Copyright (c) 2018 WSO2 Inc. (http://wso2.com) All Rights Reserved.
+~
+~ Licensed under the Apache License, Version 2.0 (the "License");
+~ you may not use this file except in compliance with the License.
+~ You may obtain a copy of the License at
+~
+~ http://www.apache.org/licenses/LICENSE-2.0
+~
+~ Unless required by applicable law or agreed to in writing, software
+~ distributed under the License is distributed on an "AS IS" BASIS,
+~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~ See the License for the specific language governing permissions and
+~ limitations under the License.
+-->
+<html>
+
+<head>
+  <title>WSO2-IAM OIDC App</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+
+<body class="login-dispatch">
+  <section id="login">
+    <div class="container">
+      <div class="row">
+	<div class="col-md-4 col-md-offset-4">
+          <div class="form-wrap-header">
+	    <img src="img/logo.png" class="center-block pickup-logo-login" alt="pickup-logo" />
+            <span class="app-title center-block">WSO2-IAM OIDC Public Client</span>
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-md-4 col-md-offset-4">
+          <div class="form-wrap">
+            <!--button class="btn btn-custom btn-lg btn-block custom-primary add-margin-top-4x" id="fetchConfig">Fetch Configuration</button-->
+            <button class="btn btn-custom btn-lg btn-block custom-primary add-margin-top-4x" id="authorize">Login</button>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <footer id="footer" class="login-footer">
+    <div class="container">
+      <div class="row">
+        <div class="col-md-12">
+          <span class="text-muted">Copyright &copy;  <a href="http://wso2.com/" target="_blank" class="wso2-logo-link"><img src="img/wso2-light.svg" class="wso2-logo" alt="wso2-logo" /></a> &nbsp;2018</span>
+        </div>
+      </div>
+    </div>
+  </footer>
+
+  <!-- extras -->
+  <!-- Bootstrap Core CSS -->
+  <link href="libs/bootstrap_3.3.7/css/bootstrap.min.css" rel="stylesheet">
+
+  <!-- Custom CSS -->
+  <link rel="stylesheet" href="./css/app.css" />
+
+  <!-- Custom Fonts -->
+  <link href="libs/font-awesome_4.3.0/css/font-awesome.min.css" rel="stylesheet" type="text/css">
+  <link href="css/font.css" rel="stylesheet" type="text/css">
+  <!-- scripts -->
+  <script src="js/jquery-3.2.1.min.js"></script>
+  <!-- Bootstrap Core JavaScript -->
+  <script src="libs/bootstrap_3.3.7/js/bootstrap.min.js"></script>
+  <script src="./js/bundle.js"></script>
+  <script type="text/javascript">
+
+    var application = null;
+    var init = function () {
```

Comment:
Since this is re used in all the html files, can we add this to a common place and reuse.

Reply:
done, added app/js/app.js file with a separate JSON config object